.env.test

```diff
@@ -1,4 +1,4 @@
-MONGO_URL=mongodb://root:root@localhost:27017/horondi?authSource=admin
+MONGO_URL=mongodb://localhost:27017/?readPreference=primary&appname=MongoDB%20Compass&ssl=false
```

Comment:
change this line to MONGO_URL=mongodb://root:root@localhost:27017/horondi?authSource=admin

Reply:
Done